tsconfig.client.json

```diff
@@ -5,7 +5,7 @@
         "allowJs": false,
         "outDir": "./dist/",
         "sourceMap": true,
-        "module": "commonjs",
+        "module": "ES6",
         "moduleResolution": "node",
         "jsx": "react",
         "target": "es5",
```

Comment:
Actually, setting the target to `es6` would be another very much viable option to get the bundlesize down by around another 150KB. The site is utterly broken on IE11 anyway, and all other browsers should support ES6. Maybe you can check the browser usage numbers we get so we can make a decision on this?

Reply:
+1 for targeting ES6.

I checked but couldn't find out how many of our visitors are using browsers/devices that only support ES5, but I would expect it to be <1%. The site also works decently well with Javascript disabled, so even for those folks they still have an option.

I would say we bump the target and see if we get any complaints (I don't expect we would).